.github/workflows/publish.yml

```diff
@@ -71,31 +75,42 @@ jobs:
         working-directory: ./
         run: ./bash/docker_container_setup.sh
 
-      - name: Download Metadata
+      - name: Determine Published Version
         run: |
-          python3 -m dcpy.connectors.edm.publishing download_file \
-            --filepath build_metadata.json \
+          published_version=$(python3 -m dcpy.connectors.edm.publishing determine_publish_version \
```

Comment:
I don't love this - I know that you you wrote this function as part of this PR so you know what went into it, but if anything else gets printed by python during a command like this (logging, etc) it ends up in the variable

Reply:
Maybe the publish cli command could have a flag to save the metadata locally? That would sort of make sense, since that's where we potentially "lose" the version (when we patch it at least), so that would be a sensible time to make a local record of where the draft is being pushed to. And then we can just grab the version from the saved metadata rather than havin the additional call to download it

Reply:
Curious on your thoughts

Reply:
yeah, I think I steered @sf-dcp a little wrong here. Writing to a file (either metadata) or just a textfile specifically for this seems like the way to go.